src/rudder-by-example/modules/ROOT/pages/policies/policies-configuration-with-vault-plugin.adoc

```diff
@@ -0,0 +1,141 @@
+
+= Configuration policies management in Rudder using Vault plugin
```

Comment:
Maybe something like "Using secrets in configuration policies" as we want to address the use-case more than the tool that is used.